components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/CacheBackedApplicationDAO.java

```diff
@@ -550,6 +603,16 @@ private void addAppBasicInfoToCache(ApplicationBasicInfo appBasicInfo, String te
         appBasicInfoCacheByName.addToCache(nameKey, entry, tenantDomain);
     }
 
+    private void addTrustedAppsToCache(List<TrustedApp> trustedApps, String platformType) {
```

Comment:
lets switch the two params so that the key (platformType) is 1st and the value (trustedApps) is second